specs/stein/approved/amd-sev-libvirt-support.rst

```diff
@@ -0,0 +1,367 @@
+..
+ This work is licensed under a Creative Commons Attribution 3.0 Unported
+ License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+
+====================================================
+libvirt driver launching AMD SEV-encrypted instances
+====================================================
+
+https://blueprints.launchpad.net/nova/+spec/amd-sev
+
+This spec proposes a minimal set of work required in order for nova's
+libvirt driver to support launching of KVM instances which are
+encrypted using `AMD's SEV (Secure Encrypted Virtualization)
+technology <https://developer.amd.com/sev/>`_.
+
+
+Problem description
+===================
+
+Cloud customers are greatly concerned about Spectre, Meltdown, and
```

Comment:
Again, I don't think its proven that SEV protects from cache sidechannel attacks, because the encryption/decryption is not bypassing caching. as such this statement  is unnecessary and potentially wrong. we don't need to do unnecessary marketing against competing vendors like AMD or intel (some of which claim potentially more complete competing technologies).

Reply:
OK, I see what you are saying. Some similar discussion [here](https://www.reddit.com/r/Amd/comments/7of4d2/eli5_is_amds_secure_encrypted_virtualization_sev/).  Reworded in next commit.